packages/owl-bot/src/database.ts

```diff
@@ -207,3 +208,29 @@ export class FirestoreConfigsStore implements ConfigsStore {
     return result;
   }
 }
+
+export class FirestoreCopyStateStore implements CopyStateStore {
```

Comment:
We use a mocked `CopyStateStore` in tests. Is there a way to test this behavior because it could potentially be complicated queries? Is there possibly a firestore emulator or can we look into an integration test?

Reply:
Added a system-test, which I confirmed passes when run locally.